### PR TITLE
Angular config subgenerator and test added

### PR DESCRIPTION
**Pull request by UndeadBaneGitHub**

### CODE REVIEW

**Comment by codydaig** [comment]
LGTM

@ilanbiala LGTY?

File: angular-config/index.js [hunk 1]
```diff
@@ -0,0 +1,58 @@
...
+
+        if (stat.isDirectory()) {
+          prompts[0].choices.push({
+            value: folder,
+            name: folder
+          });
+        }
+      });
+    }
+
```
Comment (ilanbiala) [inline angular-config/index.js]: 1 less newline

**Comment by ilanbiala** [comment]
LGTM other than that extra newline.

**Comment by codydaig** [comment]
@UndeadBaneGitHub Can you remove the extra new line so we can merge this in? Thanks!

**Comment by UndeadBaneGitHub** [comment]
Yep, in couple of hours I will remove it and fix the remaining three PRs as
well.
Commits should be squashe as they used to be, correct?

**Comment by codydaig** [comment]
Yup!

**Comment by UndeadBaneGitHub** [comment]
Fixed and squashed!

**Comment by codydaig** [comment]
LGTM

**Pull request merged**